src/js/views/globals/search/patient-search_views.js

```diff
@@ -42,11 +42,34 @@ const PatientSearchPicklist = Picklist.extend({
   itemTemplate: hbs`
     {{matchText text search}}{{~ remove_whitespace ~}}
     <span class="patient-search__picklist-item-meta">{{formatDateTime birth_date "MM/DD/YYYY"}}</span>
+    {{#each identifiers}}<span class="patient-search__picklist-item-meta">{{this}}</span>{{/each}}
   `,
   itemTemplateContext() {
+    // REMOVE: for sandbox testing purposes only
+    // this.model.set(
+    //  'identifiers',
+    //  [{ 'type': 'PID', 'value': '0123456789' }, { 'type': 'SSN', 'value': '321-45-6789' }, { 'type': 'XID', 'value': '9876543210' }],
+    // );
+
+    const settings = this.state.get('settings');
+    const settingsIdentifiers = settings ? settings.result_identifiers : [];
+
+    const patientIdentifiers = this.model.get('identifiers');
+
+    const identifiers = [];
+
+    each(settingsIdentifiers, settingsIdentifier => {
+      const identifierToShow = find(patientIdentifiers, patientIdentifier => {
+        return settingsIdentifier.toUpperCase() === patientIdentifier.type.toUpperCase();
+      });
+
+      if (identifierToShow) identifiers.push(identifierToShow.value);
```

Comment:
I think we'll likely need to display some form of empty dash if the patient doesn't have the identifier.  Mostly because I don't think we can guarantee that ids are formatted differently.  So we can probably use a map

Reply:
Sounds good, will switch it to a `map()`.

For the dash, do we want to set that in the view (e.g. `identifierToShow.value ? identifierToShow.value : '-'`)? Or set that via CSS like we do for empty widget values [here](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/scss/modules/widgets.scss).

My inclination is to do it via CSS since that's what we do elsewhere. But I feel like both would work.

Reply:
css seems fine.. otherwise.. probably and `if/else` in the template